test/parallel/test-runner-mock-timers.js

```diff
@@ -161,12 +182,14 @@ describe('Mock Timers Test Suite', () => {
         const now = Date.now();
         const timeout = 2;
         const expected = () => now - timeout;
-        global.setTimeout(common.mustCall(() => {
-          assert.strictEqual(now - timeout, expected());
-          done();
-        }), timeout);
+        global.setTimeout(
+          common.mustCall(() => {
```

Comment:
only keep what was changed (probably prettier is enabled there)

Reply:
I'll check if it's disabled here (should be, I think it's another env thing)

Reply:
Found it, already disabled, thanks for pointing!